models.py

```diff
@@ -1324,6 +1329,13 @@ class PayDbAdminTransactionsView(RestrictedModelView):
     column_export_list = ('sender', 'recipient', 'token', 'date', 'action', 'amount', 'attachment')
     column_formatters_export = {'amount': format_amount_text}
 
+    def is_accessible(self):
+        return (current_user.is_active and
+                current_user.is_authenticated and
+                (current_user.has_role(Role.ROLE_ADMIN) or
+                current_user.has_role(Role.ROLE_AUTHORIZER) or
```

Comment:
This needs to be here. Because of the access to the "DASHBOARD". Without this, there's no way for the "Authorizer" to look at the dashboard.

They can only view, they will not be able to change anything in the PayDbTransaction table.

Reply:
maybe we should get rid of authorizer and replace with finance role

Reply:
making more changes and i got a feeling that this is more than just changing the name, would require a bit of change. this will potentially break something before the demo.... its not ideal.

Reply:
also as said in the other comment, this only gives the authorizer a new "read-only" view to one of the admin tabs(Premio Transaction".

Reply:
authorizer view:
![image](https://user-images.githubusercontent.com/47799652/130525988-acef433b-b1c5-45c4-89b5-6a1eddf0c74b.png)

finance view:
![image](https://user-images.githubusercontent.com/47799652/130526139-9d4f3581-45aa-4219-ac62-3a40c1856471.png)

if we really need to not even show the authorizer role the admin tab at all, its probably better to investigate giving them no url links in the dashboard. but can still see the dashboard... but if that is the case, we might as well not give them dashboard access at all and just give them flask-admin. which i'm sure they'll keep asking someone with access, ie. admin roles, to see if there is enough funds for rewards.

Reply:
removed the authorizer access to dashboard so its only showing the admin page when the user login.

also noticed that the authorizer can create rewards as well as the finance role. Fixed on my latest push